cmdutil/debug/debug.go

```diff
@@ -85,37 +85,28 @@ type PProfServer struct {
 	pprofServer *http.Server
 }
 
-// ProfileConfig holds the configuration for the pprof server.
-type PProfServerConfig struct {
-	Addr                 string
-	MutexProfileFraction int
-}
+// NewPProfServer sets up a pprof server with configurable profiling types and returns a PProfServer instance.
+func NewPProfServer(l logrus.FieldLogger, pprofConfig *PProf) *PProfServer {
```

Comment:
Why do we need to create a new server. Why can't this be part of the `debug.Server`?